src/quo2/components/wallet/transaction_summary/component_spec.cljs

```diff
@@ -0,0 +1,8 @@
+(ns quo2.components.wallet.transaction-summary.component-spec
+  (:require [quo2.components.wallet.transaction-summary.view :as transaction-summary]
+            [test-helpers.component :as h]))
+
+(h/describe "Transaction summary"
+  (h/test "default render"
+    (h/render [transaction-summary/view {}])
+    (h/is-truthy (h/query-by-label-text :transaction-summary))))
```

Comment:
this feels like a component that might benefit from a few more tests. e.g we can check if `context` tag exists etc so that we catch cross dependencies between components. wdyt? 🤔